packages/builders/scripts/vats/terminate-original-stkATOM-USD-price-feed-governor.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @file Terminate v110 zcf-b1-9f877-stkATOM-USD_price_feed-governor.
+ */
+
+import { E } from '@endo/far';
+
+/// <reference types="@agoric/vats/src/core/types-ambient"/>
+
+/**
+ * @param {BootstrapPowers} powers
+ */
+export const terminateOriginalStkATOMPriceFeedGovernor = async ({
+  consume: { board, governedContractKits },
```

Comment:
re "No excess authority is requested" -- `governedContractKits` is access to the admin facets of many governed contracts other than the one we're working with here, so strictly speaking, that's excess authority. It's not an unusual / unreasonable amount, given available facilities. But the fact that "we must be very careful about targeting the correct vat" shows that things aren't optimal. If we had only the least authority needed, we wouldn't be able to target any other vats.

Reply:
Good observation; updated.